website/docs/feature-flag-tutorials/react/implementing-feature-flags.md

```diff
@@ -23,6 +23,11 @@ Along the way, you will:
 6. [Verify the toggle experience](#6-verify-the-toggle-experience)
 
 
+Watch the video tutorial and follow along with the code from this documentation.
+
+[![Video tutorial of how to use feature flags in React](https://img.youtube.com/vi/-VzI0wqLDuw/hqdefault.jpg)](https://www.youtube.com/watch?v=-VzI0wqLDuw)
+
```

Comment:
You probably want to use the "video content" component for this. That'll blow the video up to take up the full width of the content and let you play it on the docs instead of linking out to YouTube. 

It'd look like this:

```suggestion
import VideoContent from '@site/src/components/VideoContent.jsx';

<VideoContent videoUrls={["https://www.youtube.com/embed/-VzI0wqLDuw?si=cxLojllkIrZD8sf5"]}/>
```

Note that the import is probably better at the top of the file (just after the frontmatter), but it's fine to keep it with the usage as well.

Oh, and if you ever want to list mulitple videos, you can just keep adding youtube embed links into the `videoUrls` array.